qutip/tests/test_progressbar.py

```diff
@@ -0,0 +1,34 @@
+from qutip.ui.progressbar import progress_bars
+import pytest
+import time
+
+
+# We offer multiple alias for each progressbar.
+names = []
+bars = []
+for alias, bar in progress_bars.items():
+    if bar not in bars:
+        names.append(alias)
+        bars.append(bar)
+
+
+@pytest.mark.parametrize("pbar", names)
+def test_progressbar(pbar, capsys):
+    N = 5
+
+    try:
+        bar = progress_bars[pbar](N)
+    except ImportError:
+        # ipython or tqdm could be missing
+        pytest.skip(reason="module not available")
```

Comment:
We should probably list the progress bars explicitly and mark which ones to skip. Otherwise this will skip tests if something fails to import and it might take a long time to realize.